src/platform/nrfconnect/wifi/ConnectivityManagerImplWiFi.cpp

```diff
@@ -173,6 +173,26 @@ void ConnectivityManagerImplWiFi::OnStationDisconnected()
     }
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+CHIP_ERROR ConnectivityManagerImplWiFi::_GetSEDIntervalsConfig(ConnectivityManager::SEDIntervalsConfig & SEDIntervalsConfig)
+{
+    // Wi-Fi uses legacy power save mode that has fixed inactivity interval
+    SEDIntervalsConfig.ActiveIntervalMS = chip::System::Clock::Milliseconds32(WiFiManager::Instance().kDefaultDTIMInterval *
```

Comment:
```suggestion
    SEDIntervalsConfig.ActiveIntervalMS = chip::System::Clock::Milliseconds32(WiFiManager::kDefaultDTIMInterval *
```
etc? These are static.